### PR TITLE
Add config for discuss.k8s.io prototype

### PR DESCRIPTION
**Pull request by castrojo**
See https://github.com/kubernetes/community/blob/master/keps/sig-contributor-experience/0007-20180403-community-forum.md for more information.

### CODE REVIEW

**Comment by thockin** [comment]
Do we still need this, now that we have DNS in place?

File: k8s.io/configmap-nginx.yaml [hunk 1]
```diff
@@ -187,6 +187,14 @@ data:
           rewrite ^/(.*)$                  https://storage.googleapis.com/kubernetes-release/$1 redirect;
         }
       }
+      server {
+        server_name discuss.kubernetes.io discuss.k8s.io;
+        listen 80;
+        listen 443 ssl;
```
Comment (zparnold) [inline k8s.io/configmap-nginx.yaml]: In order for SSL to work in nginx without browsers complaining you'll need to specify where the cert files are

Comment (zparnold) [reply k8s.io/configmap-nginx.yaml]: Something like:
```
listen   443;

ssl    on;
ssl_certificate    /etc/ssl/your_domain_name.pem; (or bundle.crt)
ssl_certificate_key    /etc/ssl/your_domain_name.key;
```

Comment (zparnold) [reply k8s.io/configmap-nginx.yaml]: Ignore me. I didn't read further down the file. 😄 ssl is configured elsewhere similarly

**Comment by zparnold** [comment]
@thockin I would argue that we do, so we have control over upgrading the connection to SSL before forwarding to discourse. But additionally we could simply ask the lovely folks at discourse whether or not there is a way to configure SSL upgrades on their side to retain the DNS masking

**Comment by ixdy** [comment]
I think what @thockin is trying to say (and I agree) is that none of this will have any effect, since discuss.k8s.io (and discuss.kubernetes.io) resolve to kubernetes.hosted-by-discourse.com, not the nginx server we run.

**Comment by ixdy** [comment]
It looks like it might be possible to configure discourse to force https, though: https://docs.discourse.org/#tag/Site-Settings-Security

**Comment by castrojo** [comment]
OK there doesn't appear to be a setting that's exposed to me in the UI, so I'll ask them to see if they can force https on their end. Thanks for the feedback everyone!

**Comment by thockin** [comment]
If they can't, we can intermediate but it's better if we don't need to.

**Comment by castrojo** [comment]
They've enabled forced https for us, so we're good to go!

**Pull request closed without merging**